src/routes/docs/tutorials/vue-auth/step-1/+page.markdoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: tutorial
+title: Authentication with Vue
+description: Add Authentication to a Vue project using Appwrite.
+step: 1
+difficulty: medium
+readtime: 40
+---
+## Appwrite: The backend you'll never have to build or maintain.
```

Comment:
Please follow convention from other tutorials for what should be in this first page.